app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardGeneralStep/MappingWizardGeneralStep.js

```diff
@@ -3,6 +3,7 @@ import { Field } from 'redux-form';
 import { required } from 'redux-form-validators';
 import { Form } from 'patternfly-react';
 import { FormField } from '../../../../../common/forms/FormField';
+import { validation } from '../../../../../../../../common/constants'; // Oh my
```

Comment:
these can be eliminated by using "shared module resolution". I missed that initially... feel free to file a "code improvement" issue and we can eliminate those in the future...
https://gist.github.com/ryanflorence/daafb1e3cb8ad740b346#shared-module-resolution

Reply:
Good point. Since this `../../../../` import pattern is already spread around this repo, let's not block this PR on it. Code improvement issue sounds great, i'll open one.

Reply:
By the way @priley86 I did open an issue for this: https://github.com/priley86/miq_v2v_ui_plugin/issues/270